src/iGameGUI.c

```diff
@@ -78,24 +79,24 @@ static void flagMenuItem(struct NewMenu *, APTR, UWORD);
 
 static struct NewMenu MenuMainWin[] =
 {
-	{ NM_TITLE, STR_ID(MSG_MNlabel2Actions)							, 0 ,0 ,0			,(APTR)MENU_ACTIONS },
-	{ NM_ITEM ,  STR_ID(MSG_MNlabelScan)							,"R",0 ,0			,(APTR)MENU_SCAN },
-	{ NM_ITEM ,  STR_ID(MSG_MNMainAddnonWHDLoadgame)				,"A",0 ,0			,(APTR)MENU_ADDGAME },
-	{ NM_ITEM ,  STR_ID(MSG_MNMainMenuShowHidehiddenentries)		, 0 ,TICK ,0		,(APTR)MENU_SHOWHIDDEN },
-	{ NM_ITEM ,  NM_BARLABEL										, 0 ,0 ,0			,(APTR)0 },
-	{ NM_ITEM ,  STR_ID(MSG_MNMainAbout)							,"?",0 ,0			,(APTR)MENU_ABOUT },
-	{ NM_ITEM ,  NM_BARLABEL										, 0 ,0 ,0			,(APTR)0 },
-	{ NM_ITEM ,  STR_ID(MSG_MNMainQuit)								,"Q",0 ,0			,(APTR)MENU_QUIT },
-
-	{ NM_TITLE, STR_ID(MSG_MNlabel2Game)							, 0 ,0 ,0			,(APTR)MENU_GAME },
-	{ NM_ITEM ,  STR_ID(MSG_MNMainProperties)						,"P",0 ,0			,(APTR)MENU_GAMEPROPERTIES },
-	{ NM_ITEM ,  STR_ID(MSG_MNMainOpenCurrentDir)					,"D",0 ,0			,(APTR)MENU_GAMEFOLDER },
-
-	{ NM_TITLE, STR_ID(MSG_MNMainPreferences)						, 0 ,0 ,0			,(APTR)MENU_PREFERENCES },
-	{ NM_ITEM ,  STR_ID(MSG_MNMainiGameSettings)					, 0 ,0 ,0			,(APTR)MENU_SETTINGS },
-	{ NM_ITEM ,  STR_ID(MSG_MNlabel2GameRepositories)				, 0 ,0 ,0			,(APTR)MENU_REPOSITORIES },
-	{ NM_ITEM ,  NM_BARLABEL										, 0 ,0 ,0			,(APTR)0 },
-	{ NM_ITEM ,  STR_ID(MSG_MNMainMUISettings)						, 0 ,0 ,0			,(APTR)MENU_MUISETTINGS },
+	{ NM_TITLE, STR_ID(MSG_MNlabel2Actions), 0 ,0 ,0, (APTR)MENU_ACTIONS                        },
+	{ NM_ITEM ,  STR_ID(MSG_MNlabelScan)                    ,"R",0    ,0, (APTR)MENU_SCAN       },
+	{ NM_ITEM ,  STR_ID(MSG_MNMainAddnonWHDLoadgame)        ,"A",0    ,0, (APTR)MENU_ADDGAME    },
+	{ NM_ITEM ,  STR_ID(MSG_MNMainMenuShowHidehiddenentries), 0 ,TICK ,0, (APTR)MENU_SHOWHIDDEN },
+	{ NM_ITEM ,  NM_BARLABEL, 0 ,0 ,0, (APTR)0                                                  },
+	{ NM_ITEM ,  STR_ID(MSG_MNMainAbout)                    ,"?",0    ,0, (APTR)MENU_ABOUT      },
+	{ NM_ITEM ,  NM_BARLABEL, 0 ,0 ,0, (APTR)0                                                  },
+	{ NM_ITEM ,  STR_ID(MSG_MNMainQuit)                     ,"Q",0    ,0, (APTR)MENU_QUIT       },
+
+	{ NM_TITLE, STR_ID(MSG_MNlabel2Game), 0 ,0 ,0, (APTR)MENU_GAME                    },
+	{ NM_ITEM ,  STR_ID(MSG_MNMainProperties)    ,"P",0 ,0, (APTR)MENU_GAMEPROPERTIES },
+	{ NM_ITEM ,  STR_ID(MSG_MNMainOpenCurrentDir),"D",0 ,0, (APTR)MENU_GAMEFOLDER     },
+
+	{ NM_TITLE, STR_ID(MSG_MNMainPreferences), 0 ,0 ,0, (APTR)MENU_PREFERENCES          },
+	{ NM_ITEM ,  STR_ID(MSG_MNMainiGameSettings)     , 0 ,0 ,0, (APTR)MENU_SETTINGS     },
+	{ NM_ITEM ,  STR_ID(MSG_MNlabel2GameRepositories), 0 ,0 ,0, (APTR)MENU_REPOSITORIES },
+	{ NM_ITEM ,  NM_BARLABEL, 0 ,0 ,0, (APTR)0                                          },
+	{ NM_ITEM ,  STR_ID(MSG_MNMainMUISettings)       , 0 ,0 ,0, (APTR)MENU_MUISETTINGS  },
```

Comment:
There is no reason to change the tabs and spaces this way.

Reply:
no problem, just wanted to make it more (my style) human readable.